newdle/client/src/client.js

```diff
@@ -141,6 +141,14 @@ class Client {
     return this._request(flask`api.create_newdle`(), params);
   }
 
+  editNewdle(code, attrs) {
```

Comment:
Let's rename + make `setFinalDate` generic and turn `setFinalDate` into a shortcut calling the generic one with just the final_dt update

Reply:
Alright. With renaming, you meant the same thing or also changing `editNewdle` to something as `updateNewdle`?

Reply:
basically having one `updateNewdle` function where you pass the payload for the PATCH, and then call that one from `setFinalDate`